### ISSUE TITLE
Unable to send text to edit text element by accessibility ID from Jetpack compose

### ISSUE THREAD

**Issue by stowy** [appium/appium-uiautomator2-server#577]
I set the accessibility of a text field as follows:

```
                    OutlineTextField(
                        value = email,
                        onValueChange = viewModel::onEmailChange,
                        keyboardOptions = KeyboardOptions(
                            keyboardType = KeyboardType.Email,
                            imeAction = ImeAction.Next
                        ),
                        modifier = Modifier
                            .fillMaxWidth()
                            .focusRequester(focusRequester)
                            .semantics(mergeDescendants = true) {
                                testTag = "email";
                                testTagsAsResourceId = true},
                    )
```

It says the element cannot have the value set, eg see attached screenshot

<img width="1711" alt="Screenshot 2023-12-04 at 4 55 04 pm" src="https://github.com/appium/appium-uiautomator2-server/assets/5924628/72f32711-8cfa-4289-b42e-4eab155a28e2">

If you click on the edit text above and send it passes the text

I'm using the robot library  to send the input text which find field by accessibility Id.
